### DISCUSSION TITLE
代理设置后无法保存

### DISCUSSION THREAD

**Discussion by LiMingHuaGit** [ZuodaoTech/everyone-can-use-english#564] [Q&A]
代理设置后无法保存，每次保存后重新打开设置或重启软件代理配置即会失效。mac m1pro

**Comment by an-lee** [comment]
好像确实是有 bug

**Reply by ruruqui** [reply]
我也是这样

**Comment by an-lee** [comment]
Finished in [v0.2.4](https://github.com/xiaolai/everyone-can-use-english/releases/tag/v0.2.4)
